seed/static/seed/partials/data_upload_modal.html

```diff
@@ -233,7 +233,7 @@ <h4 class="modal-title" ng-switch-when="16" translate>PM_METER_IMPORT_RESULTS</h
             </div>
             <div class="alert alert-{$ step_10_style $}">
                 <span ng-show="step_10_error_message">{$ step_10_error_message | translate $}</span>
-                <span ng-hide="step_10_error_message" translate="COULD_NOT_LOCATE_MATCHES"></span>
+                <span ng-hide="step_10_error_message" translate="IMPORT_ANOTHER_QUESTION"></span>
```

Comment:
I'm not sure what the previous logic was, but I completely removed the "SEED could not locate any existing matches" text. 

Given that the summary in the lines above this include information on whether there were matches, this seemed like a reasonable approach.